modules/java-modules/http/src/main/java/org/syslog_ng/http/HTTPDestination.java

```diff
@@ -53,7 +53,7 @@ public HTTPDestination(long handle) {
 
     @Override
     public String getNameByUniqOptions() {
-        return String.format("HTTPDestination,%s", options.getURL());
+        return String.format("HTTPDestination,%s", "http://localhost:80/foo/bar/");
```

Comment:
If you will have two HTTP destinations in the same config then these options are not unique (this is important because this name is used as a unique key eg. in statistics).